habitat/sims/rearrangement/__init__.py

```diff
@@ -0,0 +1,26 @@
+from habitat.core.registry import registry
+from habitat.core.simulator import Simulator
+
+
+def _try_register_rearrangement_sim():
+    try:
+        import habitat_sim  # noqa: F401
+
+        has_habitat_sim = True
+    except ImportError as e:
+        has_habitat_sim = False
+        habitat_sim_import_error = e
+
+    if has_habitat_sim:
+        from habitat.sims.rearrangement.actions import (  # noqa: F401
```

Comment:
You don't need to do the if else check, you can just put these imports into the try as they will all nicely all fail under one control flow and remove the need for the `has_habitat_sim` variable.